src/lib/y2network/sysconfig/config_writer.rb

```diff
@@ -36,40 +36,50 @@ module Sysconfig
     class ConfigWriter
       include Yast::Logger
 
+      # @return [Array<Symbol>] The different sections handled by the writer
+      SECTIONS = [:routing, :drivers, :interfaces, :connections, :dns, :hostname].freeze
+
       # Writes the configuration into YaST network related modules
       #
       # @param config     [Y2Network::Config] Configuration to write
       # @param old_config [Y2Network::Config] Old configuration
-      def write(config, old_config = nil)
+      def write(config, old_config = nil, sections: :all)
```

Comment:
same here, please document sections parameter and ideally reference also SECTIONS constant from it